src/devtools/client/debugger/src/reducers/pause.js

```diff
@@ -685,4 +692,23 @@ export function getPausePreviewLocation(state: State) {
   return state.pause.previewLocation;
 }
 
+export function getResumePoint(state, type) {
+  const thread = getCurrentThread(state);
+  const framePoints = getFramePositions(state, thread)?.positions.map(p => p.point);
```

Comment:
This should await until the positions are loaded so behavior is consistent even if things haven't finished loading yet.

Reply:
good call